### PR TITLE
GCE module: allow to keep instance boot disk after instance deletion

### PR DESCRIPTION
**Pull request by nabam**
Useful to create images from boot disks with gce_img

### CODE REVIEW

**Comment by nabam** [comment]
@erjohnso please take a look at this pull request also

File: cloud/google/gce.py [hunk 1]
```diff
@@ -129,6 +129,12 @@
     required: false
     default: "ephemeral"
     aliases: []
+  disk_auto_delete:
```
Comment (erjohnso) [inline cloud/google/gce.py:132]: Should we have a `version_added` here?

**Comment by erjohnso** [comment]
One quick suggestion, but otherwise looks great, thanks for this @nabam.

File: cloud/google/gce.py (hunk 1 above)
Comment (nabam) [inline cloud/google/gce.py:132]: @erjohnso done

**Comment by erjohnso** [comment]
Thanks @nabam, lgtm.

**Pull request merged**